cfr.yaml

```diff
@@ -133,4 +133,4 @@
   # For versions 89+ we trigger the message at startup and want to ignore the
   # newtab trigger.
   targeting:
-    "((firefoxVersion >= 87 && firefoxVersion < 89) || (firefoxVersion >= 89 && source == 'startup')) && !isDefaultBrowser && !'browser.shell.checkDefaultBrowser'|preferenceValue && isMajorUpgrade != true && platformName != 'linux' && ((currentDate|date - profileAgeCreated) / 604800000) >= 5 && !activeNotifications"
+    "((firefoxVersion >= 87 && firefoxVersion < 89) || (firefoxVersion >= 89 && source == 'startup')) && !isDefaultBrowser && !'browser.shell.checkDefaultBrowser'|preferenceValue && isMajorUpgrade != true && platformName != 'linux' && ((currentDate|date - profileAgeCreated) / 604800000) >= 5 && !activeNotifications && 'browser.newtabpage.activity-stream.asrouter.userprefs.cfr.features'|preferenceValue"
```

Comment:
Do we want to update the `profileAgeCreated > 5 weeks` also check `< 15 weeks` to get the "5 weeks and 10 weeks after installation" behavior? The current patch works for new users but existing longtime users could still see it as it's "10 weeks after we launched". Unclear if there should be a different behavior for new/existing.